src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/draft/DraftAdjudicationController.kt

```diff
@@ -160,7 +161,7 @@ class DraftAdjudicationController(
     )
 
   @PostMapping
-  @PreAuthorize("hasAuthority('SCOPE_write')")
+  @PreAuthorize("hasRole('ADJUDICATIONS_REVIEWER') and hasAuthority('SCOPE_write')")
```

Comment:
For each `SCOPE_write` (except for migration) I added the role `ADJUDICATIONS_REVIEWER `

Reply:
This was incorrect :) updated with https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467/commits/ec613a63d5ad153abaf7f88ccfe88a77d2fa935f